### PR TITLE
Adds a living time config to the panic bunker

### PR DESCRIPTION
**Pull request by Sinestia**
## About The Pull Request

Ports https://github.com/tgstation/tgstation/pull/54341
Requested by higher-ups

This allows you to set a minimum living time to bypass the panic bunker, set in minutes. So 60 = you must have 1 hour of played living time to connect. 

This can be configured both in config and in-game through the button, however the message does not display to the blocked. From what I was told apparently Bee's panic bunker is just a bit Sus (the redirect never showed up either,) but the important functionality is here at least. 

This was tested by editing an alt into a local database and connecting, so it should work. 
## Why It's Good For The Game

Power to the admins or something. 

## Changelog
:cl:LemonInTheDark, Port By Froststahr. 
admin: The panic bunker allows for a minimum time to be set, blocking players who lack the required amount of living played time from accessing the game. 
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by Crossedfall** [comment]
Well, it's part of what I wanted. 

If you could include:

https://github.com/tgstation/tgstation/pull/59530
https://github.com/tgstation/tgstation/pull/54465

That would be incredibly helpful. ❤️ 

Going to TM this as is though so we can at least get the living time config in place.

**Review by PowerfulBacon — DISMISSED** [review]
config changes

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -595,11 +595,19 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
...
-			log_access("Failed Login: [key] - New account attempting to connect during panic bunker")
-			message_admins("<span class='adminnotice'>Failed Login: [key] - New account attempting to connect during panic bunker</span>")
-			to_chat(src, CONFIG_GET(string/panic_bunker_message))
+
+	//If we aren't an admin, and the flag is set
+	if(CONFIG_GET(flag/panic_bunker) && !holder && !GLOB.deadmins[ckey])
+		var/living_recs = CONFIG_GET(number/panic_bunker_living)
+		//Relies on pref existing, but this proc is only called after that occurs, so we're fine.
+		var/minutes = get_exp_living(pure_numeric = TRUE)
+		if(minutes <= living_recs)
```
Comment (qwertyquerty) [inline code/modules/client/client_procs.dm]: This should just be `<`

**Comment by ike709** [comment]
@Crossedfall babe apply config for me and merge por favor

**Review by Crossedfall — APPROVED** [review]
Guess I'll get the other changes ported another time. 😔

**Pull request merged**